ui/src/organizations/containers/OrgProfilePage.tsx

```diff
@@ -32,7 +32,7 @@ class OrgProfilePage extends Component<StateProps> {
             <Grid>
               <Grid.Row>
                 <Grid.Column widthXS={Columns.Twelve} widthSM={Columns.Six}>
-                  <OrgProfileTab />
+                  <OrgProfileTab orgID={org.id} orgName={org.name} />
```

Comment:
why isn't it just grabbing it's own org info from the store?

Reply:
That's a good question. Long live discreet components!!